proto/backend.proto

```diff
@@ -69,23 +113,27 @@ message DataQueryResponse {
 // HTTP style request
 //-----------------------------------------------
 
-message ResourceRequest {
-  // Plugin Configuration
-  PluginConfig config = 1;
-
+message CallResource {
+  message StringList {
+    repeated string values = 1;
+ }
 
-  // HTTP Style parameters
-  map<string,string> headers = 2; // Request Headers
-  string method = 3; // GET/POST/DELETE/etc
-  string path = 4;   // (with params?)
-  bytes body = 5;    // assume string unless header says otherwise?
-}
-
-message ResourceResponse {
-  int32 code = 1;  // Match HTTP status codes
-  map<string,string> headers = 2; // Response Headers
+  message Request {
+    PluginConfig config = 1;
+    string resourceName = 2;
+    string resourcePath = 3;
+    string method = 4;
+    string url = 5;
+    map<string,StringList> headers = 6;
```

Comment:
what is the StringList for?   multiple headers same key?

Reply:
Yeah multiple values for the same key.